core/includes/unicode.inc

```diff
@@ -319,14 +319,22 @@ function backdrop_truncate_bytes($string, $len) {
  *   that "See" alone is too short, and would then just truncate ignoring word
  *   boundaries, giving you "See myverylongurl..." (assuming you had set
  *   $add_ellipses to TRUE).
+ * @param $include_newlines
```

Comment:
```suggestion
 * @param bool $include_newlines
```

While at it, also add types to the rest of the parameters in the function?

Reply:
Thanks.